src/assets/json/productTest.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
If I understand correctly. You use the data only for testing. So I would move it into mock directory